### PR TITLE
Fizz Buzz in Scala

### LINKED ISSUE #516: Add Fizz Buzz in Perl
If this is your first time contributing, create the Fizz Buzz file in the appropriate directory and add the name of the program to the Sample Programs section of the language README. Then, create a new issue for its article like #30. Link this issue in the README and create a pull request when you're done.

### LINKED ISSUE #521: Add Fizz Buzz in Scala
If this is your first time contributing, create the Fizz Buzz file in the appropriate directory and add the name of the program to the Sample Programs section of the language README. Then, create a new issue for its article like #30. Link this issue in the README and create a pull request when you're done.

### PR DESCRIPTION
**Pull request by chris-c-thomas**
## Notes
Fizz Buzz in Scala

Fixes #521 
Fixes #516

### CODE REVIEW

**Comment by chris-c-thomas** [comment]
I have Fizz Buzz in Perl created as well but do not want to push it to my master branch until this is merged so that each PR contains a single program/article.

**Comment by jrg94** [comment]
Awesome! If you want, you can make a local branch on your repo and pull request from that.

File: archive/s/scala/README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ Welcome to Sample Programs in Scala!
 ## Sample Programs
 
 - [Hello World in Scala](https://therenegadecoder.com/code/hello-world-in-scala/)
```
Comment (jrg94) [inline archive/s/scala/README.md]: Just a general request, but could you convert all the links in this file to the format you've started (i.e. links numbered at the end of the file)? While you're there, can you alphabetize the program list? That would be awesome!

**Comment by jrg94** [comment]
Nice job! Looks great so far.

**Comment by chris-c-thomas** [comment]
@jrg94 Links Updated!

File: archive/p/perl/README.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ Welcome to Sample Programs in Perl!
 
 - [Hello World in Perl](https://therenegadecoder.com/code/hello-world-in-perl/)
```
Comment (jrg94) [inline archive/p/perl/README.md]: Looks like this pulled in your changes from Perl as well which is fine. Can you make the same README changes here? 👍

Comment (chris-c-thomas) [reply archive/p/perl/README.md]: Shoot! I didn't mean to do that, sorry. Updated the Perl links just now though.

**Pull request merged**